pkg/epp/plugins/plugins.go

```diff
@@ -22,3 +22,7 @@ type Plugin interface {
 	// Name returns the name of the plugin.
 	Name() string
 }
+
+// Handle provides plugins  set of standard data and tools to work with
+type Handle interface {
+}
```

Comment:
I went over all plugins and couldn't find any use for Handle interface.
can you explain what is the purpose of this interface?

Reply:
we have very bad experience with adding unused interfaces or fields. 
the last time led to chat completions being broken cause it doesn’t have prompt field in the request body. 

I strongly recommend to add fields/interfaces only when it’s used and only along with unit tests that cover them.

Reply:
That is why there is an empty implementation and nil is never passed...

Reply:
pointer to an empty `eppHandle` that is never used is not good example, in this case you could even pass nil and nothing would fail cause it's not used anywhere :). (don't get me wrong, I'm not suggesting to pass nil).
generally speaking, I'm not in favor of adding unused/empty place holders.
this may make sense if there is a follow up PR planned to be pushed right after this one and if it's done just to split a big PR to smaller chunks.

Reply:
The goal is to avoid changing the interface to the plugin developer if possible.

Changes in the Handle interface will the EPP code, but not the existing plugins

Reply:
do we expect *existing* plugins to get some information/control from the eppHandle?
If so, the change is inevitable.